swaap/swaap/View Controllers/SignUpViewController.swift

```diff
@@ -40,6 +40,8 @@ class SignUpViewController: UIViewController, AuthAccessor {
 				if let error = error {
 					let alertVC = UIAlertController(error: error)
 					self?.present(alertVC, animated: true)
+				} else {
+					self?.parent?.dismiss(animated: true, completion: nil)
 				}
```

Comment:
refactor to have the error condition `return`

(communicates intention more clearly)

Reply:
(no need for an else statement in that case)